run_pylint.sh

```diff
@@ -34,6 +34,7 @@ INIT_HOOK="
 AP = os.environ.get('AUTOTEST_PATH', '/usr/local/autotest')
 sys.path.append(os.path.abspath(AP + '/..'))
 sys.path.append(os.path.abspath('.'))
+sys.path.append(os.path.abspath('dockertest'))
```

Comment:
I think this problem must be related to the one I'm seeing with different namespaces in exceptions. But again, I'm starting to wonder if this is a worthwhile effort right now.